hbase/src/main/scala/akka/stream/alpakka/hbase/HBaseConverter.scala

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.hbase
+
+import org.apache.hadoop.hbase.client.Put
+import org.apache.hadoop.hbase.util.Bytes
+
+import scala.language.implicitConversions
+
+trait HBaseConverter[T] {
+  def put(t: T): Put
+}
```

Comment:
As this is just a single function, couldn't this just be a `T => Put` parameter then, rather than a specific type?